content/en/docs/concepts/services-networking/gateway.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Gateway
+content_type: concept
+description: >-
+  Make network services available by using an extensible, role-oriented, protocol-aware
+  API. Gateway is a family of API kinds that provide dynamic infrastructure provisioning
+  and advanced traffic routing for Kubernetes services.
+weight: 55
+---
+
+<!-- overview -->
+
+Make network services available by using an extensible, role-oriented, protocol-aware configuration
+mechanism. The [Gateway API][gw-api] consists of API [kinds][kinds] that extend Kubernetes to provide
+dynamic infrastructure provisioning and advanced traffic routing.
+
+<!-- body -->
+
+## Design principles
+
+The following principles shaped the design and architecture of Gateway:
+
+* __Role-oriented:__ Gateway API kinds are modeled after organizational roles that are
+  responsible for managing Kubernetes service networking:
+  * __Infrastructure Provider:__ Manages infrastructure that allows multiple isolated clusters
+    to serve multiple tenants, e.g. a cloud provider.
+  * __Cluster Operator:__ Manages clusters and is typically concerned with policies, network
+    access, application permissions, etc.
+  * __Application Developer:__ Manages an application running in a cluster and is typically
+    concerned with application-level configuration and [Service][service] composition.
+* __Portable:__ Gateway API specifications are defined as [custom resources][crds] and are supported
+  by many [implementations][implementations].
+* __Expressive:__ Gateway API kinds support functionality for common traffic routing use cases
+  such as header-based matching, traffic weighting, and others that were only possible in [Ingress][ingress]
+  by using custom annotations.
+* __Extensible:__ Gateway allows for custom resources to be linked at various layers of the API.
+  This makes granular customization possible at the appropriate places within the API structure.
+
+Refer to the full Gateway API [documentation][gw-api] for additional details of these principles.
```

Comment:
I'd list that much later in the page, maybe just before the What's next heading.